app/views/pages/get-involved.html.erb

```diff
@@ -1,55 +1,183 @@
-    </div>
-    <div class="govuk-grid-row govuk-!-padding-top-7">
-      <div class="govuk-grid-column-two-thirds">
-        <h2 class='govuk-heading-l govuk-!-margin-bottom-7'>Our partnerships</h2>
+  <%= render BannerComponent.new(
+    background_color: "purple",
+    title: t(".ambassador.heading"),
+    title_css: "govuk-heading-m",
+    title_link: nil,
+    text_blocks: [
+      t(".ambassador.body_1", redirect_link: link_to("STEM Ambassador", stem_ambassadors_url)).html_safe,
+      t(".ambassador.body_2")
+    ],
+    text_css: "govuk-body-s",
+    video: { url: ambassador_video_url },
+    video_type: :youtube,
+    media_column_size: :half,
+    media_side: :right,
+    padding: 5,
+    button: { 
+      text: t(".ambassador.link.title"),
+      link: stem_ambassadors_url,
+      class: "govuk-button button bordered-card__get-involved-button bordered-card__external-button",
+      event_action: "click",
+      event_category: t(".ambassador.link.event_category"),
+      event_label: t(".ambassador.link.event_label")
+    },
+  ) %>
+
+  <%= render BannerComponent.new(
+    title: t(".governor.heading"),
+    title_css: "govuk-heading-m",
+    title_link: nil,
+    text_blocks: [
+      t(".governor.body_1"),
+      t(".governor.body_2"),
+      t(".governor.body_3")
+    ],
+    text_css: "govuk-body-s",
+    video: { url: governor_video_url },
+    video_type: :youtube,
+    media_column_size: :half,
+    media_side: :right,
+    padding: 5,
+    button: { 
+      text: t(".governor.link.title"),
+      link: governors_and_trustees_url,
+      class: "govuk-button button get-involved_governor-button",
+      event_action: "click",
+      event_category: t(".governor.link.event_category"),
+      event_label: t(".governor.link.event_label")
+    },
+  ) %>
+
+  <div class="light-grey-bg">
+    <div class="govuk-width-container">
+      <div class="govuk-main-wrapper">
+        <div class="govuk-grid-row get-involved-banner">
+          <div class="govuk-grid-column-full light-grey-bg">
+            <h2 class="govuk-heading-m"><%= t(".donate.heading") %></h2>
+
+            <p class="govuk-body-m"><%= t(".donate.body_1") %></p>
+            <p class="govuk-body-s"><%= t(".donate.body_2") %></p>
+          
+            <div class="get-involved_get-in-touch-button-container">
+              <%= link_to t(".donate.link.title"), governors_and_trustees_url, class: "govuk-button button button--inverted get-involved_get-in-touch-button bordered-card__external-button"%>
+            </div>
+          </div>
+        </div>
       </div>
     </div>
-    <div class="govuk-grid-row">
-      <div class="govuk-grid-column-full govuk-!-margin-top-3">
-        <%= render BorderedCardsComponent.new(**(GetInvolved.partnership_cards)) %>
+  </div>
+
+  <%= render BannerWithBackgroundComponent.new(
+    background_image_css: "get-involved-background_classroom",
+    label: {
+      title: t(".impact.heading"),
+      body: t(".impact.body"),
+    }
```

Comment:
It looks like the star file is available (.icon-star-black / star-black.svg), but don't see it referenced for display?

Reply:
Perhaps ditto to the above

Reply:
Needed a server restart on my side, all good 👍